packages/cli/commands/auth.js

```diff
@@ -71,7 +71,7 @@ exports.describe = i18n(`${i18nKey}.describe`, {
 });
 
 exports.handler = async options => {
-  const { type, config: c, qa, account } = options;
+  const { type, config: c, qa, accountFlag } = options;
```

Comment:
Interesting, is the creation of the `accountFlag` something we are getting for free from `yargs`?  It looks like it since the option remained intact.  I like this better than blocking particular commands 👍 

The only thing I think might be nice to add is comment for our futures selves to remember why we are using this version instead of `account`

Reply:
Sounds good, will add comments now!